index.js

```diff
@@ -24,39 +25,40 @@ program
       }
     }
 
-    var num = numberOfResults;
-
-    if(numberOfResults === undefined) {
-      num = 3;
-    }
-
     request(options, function (err, res, body) {
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
 
       console.log('='.repeat(consoleWidth()));
 
+      if(typeof process.argv[4] !== 'undefined'){
+        if(typeof Number(process.argv[4]) === 'number'){
+          resultsToDisplay = Number(process.argv[4]);
+        }
+      }
+
```

Comment:
How are we handling negative inputs?

Reply:
what do you mean negative inputs? Like a non-zero number?

Reply:
negative numbers*